daemons/ipa-slapi-plugins/ipa-pwd-extop/prepost.c

```diff
@@ -766,7 +766,7 @@ static int ipapwd_pre_mod(Slapi_PBlock *pb)
     /* Check this is a clear text password, or refuse operation (only if we need
      * to comput other hashes */
     if (! unhashedpw && (gen_krb_keys || is_smb || is_ipant)) {
-        if ('{' == userpw[0]) {
+        if (userpw && ('{' == userpw[0]) {
```

Comment:
A closing ')' is missing.

Reply:
Looks like we have a dependency bug in our Makefile system. ``make`` didn't fail. I had to do ``./autogen.sh && make clean`` to see the compile error.